.github/workflows/refresh_deployment.yaml

```diff
@@ -0,0 +1,64 @@
+name: Refresh EC2 FINOS Waltz deployment
+
+on:
+  workflow_dispatch:
+  #schedule:
+  # Runs everyday at 00:00. (see https://crontab.guru)
+  #- cron: '0 0 * * *'
+
+jobs:
+  build:
+    name: Trigger Juju refresh on EC2 Deployment
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Can you see if there's an already existing github action that would help us do the remote ssh bits for us? At a glance, this looks interesting: https://github.com/marketplace/actions/ssh-remote-commands . That would make our github action simpler, and we wouldn't have to reinvent the wheel.

Reply:
That one from the market place seems to work pretty well for us. I've updated our action to use it.